cpp/harvest/nacjd.cc

```diff
@@ -43,11 +43,18 @@ namespace {
 
 [[noreturn]] void Usage() {
     ::Usage(
-        "marc_title_in_file new_marc_title_out_file [count_of_rows]\n"
+        "marc_title_in_file new_marc_title_out_file [mode] [mode_params]\n"
         "where marc_title_in_file contains also icpsr records (001 or 035a)\n"
         "these records are not processed any more\n"
         "new_marc_title_out_file contains all icpsr records not contained in input file.\n"
-        "count_of_rows is an optional parameter. If it is absent, the number of requests will be equal to 9000)");
+        "\n"
+        "marc_title_in_file new_marc_title_out_file count_of_rows number\n"
```

Comment:
this should only be
`"count_of_rows number\n"`

with a fitting description. But since i don't really understand what this mode should do and it is also not implemented yet, we should talk about it before you continue working on this mode.